test/junit/scala/collection/immutable/LazyListTest.scala

```diff
@@ -163,7 +163,7 @@ class LazyListTest {
     cyc.tail.tail.head
     assertEquals("LazyList(1, 2, 3, <not computed>)", cyc.toString)
     cyc.tail.tail.tail.head
-    assertEquals("LazyList(1, 2, 3, 4, <cycle>)", cyc.toString)
+    assertEquals("LazyList(1, 2, 3, 4, <not computed>)", cyc.toString)
```

Comment:
Could you also evaluate `cyc.tail.tail.tail.tail.head` and check that `toString` returns `"LazyList(1, 2, 3, 4, <cycle>)"`?

Reply:
sure!

Reply:
I believe that is immediately below

Reply:
Haha, perfect, I didn’t see it!